packages/data-point/data-point.d.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
this is outside the scope of data-point, please remove

Reply:
done